apps/super-rentals/.travis.yml

```diff
@@ -0,0 +1,31 @@
+---
```

Comment:
Do we still need this file?

Reply:
As far as I know, it doesn't do anything when it's not at the root of the repository, and we don't intend for it to run, so in that sense it's not needed and we could delete it. (Likewise, I think the `yarn.lock` in here doesn't actually do anything either, since we are in a workspace with its a "shared" lock file at the root.) However, since it's benign, it's probably better to leave it around, otherwise, it's will cause a conflict when we pull changes from upstream (if upstream made changes to this file but we deleted it, it will cause a merge conflict).